runbot_travis2docker/models/runbot_build.py

```diff
@@ -178,6 +178,11 @@ def get_docker_run_cmd(self):
                 '-e', ('WEBLATE_HOST=%s' %
                        build.branch_id.repo_id.weblate_url)
             ]
+            if build.branch_id.repo_id.weblate_languages:
+                wl_cmd_env += [
+                    '-e', 'LANG_ALLOWED=%s' %
```

Comment:
@moylop260 
MQT find all files `*.po` into the folder `i18n` https://github.com/Vauxoo/maintainer-quality-tools/blob/master/travis/travis_weblate.py#L111 and add all files found in https://github.com/Vauxoo/maintainer-quality-tools/blob/master/travis/travis_weblate.py#L137